itests/src/test/java/fi/metatavu/edelphi/test/mock/QueryMocker.java

```diff
@@ -0,0 +1,272 @@
+package fi.metatavu.edelphi.test.mock;
+
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import fi.metatavu.edelphi.test.mock.QueryPageMock.SettingType;
+
+public class QueryMocker extends AbstractResourceMocker {
+
+  private static Logger logger = Logger.getLogger(QueryMocker.class.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the "logger" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)